mu4/notation/internal/notationselection.h

```diff
@@ -38,17 +38,36 @@ class NotationSelection : public INotationSelection
     bool isNone() const override;
     bool isRange() const override;
 
+    bool canCopy() const override;
+    QMimeData* mimeData() const override;
+
     Element* element() const override;
     std::vector<Element*> elements() const override;
 
     SelectionRange range() const override;
```

Comment:
I think it'd be better to add rangeBoundingArea, rangeStartSegment and rangeEndSegment to SelectionRange class

Reply:
thanks!
fixed